specify/specify7
# Add search on synonyms in the QB

Fixes #752

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- open query builder and select a tree table 
- make a search on any rank using the name as a field (chose a node that has a synonym in your tree) 
- check the box search synonyms 
- run the query 
- [ ] verify that the node is being return as well as its synonym
